client/src/components/ReUsableComponents/CustomTextField.tsx

```diff
@@ -0,0 +1,56 @@
+import Box from '@mui/material/Box/Box';
+import FormControl from '@mui/material/FormControl';
+import InputLabel from '@mui/material/InputLabel';
+import TextField from '@mui/material/TextField';
+import * as React from 'react';
+
+interface Props {
+  id: string;
+  value: string;
+  name: string;
+  label: string;
+  handleChange: any;
```

Comment:
Generally not preferred to use any in TypeScript. Since we're using a controlled input here, we could do at least `(e: any) => void;` as the type of handleChange or we can make it more defined `(value: string) => void;`

But then in handleChange we'd need to do the following `onChange={(e) => handleChange(e.target.value)}`